build.gradle

```diff
@@ -28,6 +28,7 @@ apply from: rootProject.projectDir.path + '/gradle/scripts/jacoco-coveralls-supp
 
 idea.project {
   ext.languageLevel = JavaVersion.VERSION_1_7
```

Comment:
Yes, I believe that is for the Gradle 2.+ migration. However, the `ext.` prefix is only for global properties that need to be accessed project wide. It's basically a global map. `languageLevel` is specific to the configuration of the `idea` plugin, so its not a global property.

In fact, this commit probably broke the project version specification for idea: https://github.com/linkedin/gobblin/commit/15984d7b16056b00ec9aa27b0bfb789b1ddd174d